tiny-firmware/firmware/fsm_skycoin_impl.c

```diff
@@ -209,7 +213,9 @@ msgTransactionSignImpl(TransactionSign *msg, ErrCode_t (*funcConfirmTxn)(char *,
             uint8_t seckey[32] = {0};
             size_t size_address = 36;
             char address[36] = {0};
-            ErrCode_t ret = fsm_getKeyPairAtIndex(1, pubkey, seckey, NULL, msg->transactionOut[i].address_index);
+            int (*address_from_pubkey)(const uint8_t*, char*, size_t*);
+            address_from_pubkey = &skycoin_address_from_pubkey;
+            ErrCode_t ret = fsm_getKeyPairAtIndex(1, pubkey, seckey, NULL, msg->transactionOut[i].address_index, address_from_pubkey);
```

Comment:
You can omit creating `address_from_pubkey` variable and directly path `&skycoin_address_from_pubkey` to fsm_getKeyPairAtIndex.